framework/src/base/Moose.C

```diff
@@ -572,6 +575,7 @@ bool _warnings_are_errors = false;
 bool _deprecated_is_error = false;
 bool _throw_on_error = false;
 bool _throw_on_warning = false;
+int autosave_flag = 0;
```

Comment:
Let's rename this variable, we aren't using it as a flag anymore (maybe ever). It's not a boolean and it's value is significant. I'd recommend one of the following:
"unix_signal_number"
"max_signal_number"
"interrupt_signal_number"